Makefile

```diff
@@ -896,6 +896,9 @@ CPPLINT_FILES = $(filter-out $(CPPLINT_EXCLUDE), $(wildcard \
 	src/*.c \
 	src/*.cc \
 	src/*.h \
+	src/tracing/.c \
+	src/tracing/.cc \
+	src/tracing/.h \
```

Comment:
ok , sure I thought I want to keep now as a folder specific. 

I will do the changes